packages/@divvi/mobile/src/components/TokenIcon.tsx

```diff
@@ -62,6 +63,7 @@ interface Props<T extends { imageUrl?: string; symbol?: string | never; networkI
 export default function TokenIcon<
   T extends { imageUrl?: string; symbol?: string | never; networkIconUrl?: string },
 >({ token, viewStyle, testID, size = IconSize.MEDIUM, showNetworkIcon = true }: Props<T>) {
+  showNetworkIcon = showNetworkIcon && getSupportedNetworkIds().length > 1
```

Comment:
this is a bit of an assumption, but we could add one or two configs for this. E.g., one for token icons and one for tx feed icons. Thoughts?

Reply:
Looks fine for now.